docs/.eleventy.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Fichier de configuration Eleventy
+ * https://www.11ty.dev/docs/config/
+ */
+
+const pluginWebc = require("@11ty/eleventy-plugin-webc");
+
+module.exports = function(eleventyConfig) {
+
+  eleventyConfig.addPlugin(pluginWebc, {
+    // WebC Components
+    // Nous sommes explicites dans le chemin depuis lequel
+    // nous souhaitons importer.
+    components: [
+      "./src/_includes/components/**/*.webc",
+    ],
+  });
+
+  // Copier les contenus du dossier `uploads/` tels quels
+  // (sans le dossier `uploads/` dans le chemin de destination)
+  eleventyConfig.addPassthroughCopy("uploads");
```

Comment:
D'après le déploiement sur Netlify le répertoire `uploads` est conservé :

https://stylo-docs.netlify.app/uploads/images/Login-Huma-Num-V2.PNG

D'ailleurs il faut revoir les liens vers les images afin d'utiliser un chemin absolu `/images`.

Reply:
D'après la documentation il faut préciser la target: https://www.11ty.dev/docs/copy/#change-the-output-directory

Reply:
> D'après le déploiement sur Netlify le répertoire `uploads` est conservé

La documentation précédente ne nécessitait aucun build (affichage des fichiers markdown côté client avec MDwiki), c’était donc copié tel quel.

Pour des fins de clarté, je pense que c’est plus simple si le dossier `uploads/` est copié tel quel, et que les liens vers les images utilisez le chemin absolu `/uploads/images` (cela serait aussi compatible avec l’installation actuelle; les liens vers les images ne seront pas brisés lors du déploiement du nouveau site).

Reply:
On peut conserver le répertoire `uploads` mais il faudra quand même changer le chemin vers les images car actuellement le chemin est relatif à la page.

Reply:
Oui, la plupart des pages ont un lien relative `uploads/images` – j’ai mentionné cela par écrit aux autres (Roch, Giulia) qui vont se charger de l’édition de la doc!

Reply:
OK parfait 👍